front/pages/api/w/[wId]/members/[uId]/index.ts

```diff
@@ -25,10 +22,14 @@ async function handler(
   res: NextApiResponse<WithAPIErrorResponse<PostMemberResponseBody>>,
   auth: Authenticator
 ): Promise<void> {
+  const owner = auth.getNonNullableWorkspace();
+
   if (!auth.isAdmin()) {
     // Allow Dust Super User to force role for testing
     const allowForTesting =
-      isDevelopment() && auth.isDustSuperUser() && req.body.force === "true";
+      canForceUserRole(owner) &&
```

Comment:
No need to action.

I generally try to avoid these very trivial functions: rationale, the abstraction does not really save code and it's less explicit here as I need to travel to it to understand what it does...

General idea is that abstractions are not always the best path forward as it's easy to land on the side of introducing too many.

Not saying it's bad here, just communicating that I think it's on the fence :+1:

Reply:
Here it's definitely not made to save code but to have a simple way to connect the front end (show the options) and the backend (allowing the action) and keep them in sync.